src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -2401,20 +2575,37 @@ void AutomationInstrumentClipView::modEncoderButtonAction(uint8_t whichModEncode
 
 				//display only left cursor initially
 				leftPadSelectedX = 0;
-				rightPadSelectedX = kNoLastSelectedPad;
+				rightPadSelectedX = kNoSelection;
+
+				if (!playbackHandler.isEitherClockActive()) {
+					ModelStackWithAutoParam* modelStackWithParam =
+						getModelStackWithParam(modelStack, clip, clip->lastSelectedParamID, clip->lastSelectedParamKind);
+					if (modelStackWithParam && modelStackWithParam->autoParam) {
+
+						if (modelStackWithParam->getTimelineCounter()
+							== view.activeModControllableModelStack.getTimelineCounterAllowNull()) {
+
+							view.activeModControllableModelStack.paramManager->toForTimeline()->grabValuesFromPos(
+								getPosFromSquare(leftPadSelectedX), &view.activeModControllableModelStack);
+
+							displayAutomation();	
+						}
+					}
+				}
 			}
 		}
 	}
```

Comment:
code added to restore regular mod encoder button actions when in automation overview screen